frame/balances/src/lib.rs

```diff
@@ -369,6 +369,26 @@ pub mod pallet {
 			<Self as Currency<_>>::transfer(&transactor, &dest, value, KeepAlive)?;
 			Ok(().into())
 		}
+
+		/// Transfer the entire transferable balance from the caller account.
+		///
```

Comment:
maybe add a note on what `keep_alive` does and how it leads to only a partial transfer?